crates/op-rbuilder/src/builders/standard/payload.rs

```diff
@@ -362,7 +361,13 @@ impl<Txs: PayloadTxsBounds> OpBuilder<'_, Txs> {
         let block_da_limit = ctx
             .da_config
             .max_da_block_size()
-            .map(|da_size| da_size.saturating_sub(builder_tx_da_size));
+            .map(|da_size| {
+                let da_size = da_size.saturating_sub(builder_tx_da_size);
+                if da_size == 0 {
+                    error!("Builder tx da size subtraction caused max_da_block_size to be 0. No transaction would be included.");
```

Comment:
it'll be helpful for debugging to log the builder tx da size + block da size

Reply:
I think it would be cool to add the amount of da size left at the end of the block as a metric
We already have a log if da exceeded